ext/ffi/ffi.g

```diff
@@ -96,7 +96,10 @@ declarations:
 				initializer?
 				{zend_ffi_declare(name, name_len, &dcl);}
 			)*
-		)?
+		|
+			/* empty */
+			{if (common_dcl.flags & ZEND_FFI_DCL_ENUM) zend_ffi_cleanup_dcl(&common_dcl);}
```

Comment:
This looks right.

I'm still not completely sure if ``common_dcl.flags & ZEND_FFI_DCL_ENUM`` is the right condition.
Currently we may also have similar structures and they also may lead to memory leaks.
GCC allows this syntax with a warning: ``unnamed struct/union that defines no instances``.